knative-operator/deploy/crds/operator_v1alpha1_knativekafka_crd.yaml

```diff
@@ -0,0 +1,120 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: knativekafkas.operator.serverless.openshift.io
+spec:
+  group: operator.serverless.openshift.io
+  names:
+    kind: KnativeKafka
+    listKind: KnativeKafkaList
+    plural: knativekafkas
+    singular: knativekafka
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
```

Comment:
above the `properties`, wanna add something like:

https://github.com/openshift-knative/serverless-operator/blob/master/olm-catalog/serverless-operator/manifests/operator_v1alpha1_knativeeventing_crd.yaml#L28-L29

Reply:
Done, thanks